test/DynamoCoreUITests/RecordedTests.cs

```diff
@@ -684,10 +689,22 @@ protected void LoadCustomNodes(string[] customNodeFilePaths)
                 throw new InvalidOperationException(message);
             }
 
+            var geometryFactoryPath = string.Empty;
+            if (preloadGeometry && (preloader == null))
```

Comment:
You're setting preloader to null again, so you're not guaranteed to prevent multiple calls.